pallets/rad-claims/src/weights.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Centrifuge (centrifuge.io) parachain.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+
+//! Rad claims pallet's extrinsics weight information
+//! 
+//! This module must contain the weights that are calculated using
+//! runtime benchmarking. 
+//! This file will be replaced when you first run benchmarking.
+//! You should first write various benchmarking cases in the 
+//! [`benchmarking`](./benchmarking.rs) module.
```

Comment:
I think we can remove this file and come back to this later during benchmarks

Reply:
Possible of course

Reply:
I would prefer to remove this file too, as it implies at first glance, that we already have a benchmarking in place.

Reply:
Bouhhhhhhh 
![image](https://user-images.githubusercontent.com/81358014/118982366-a5e07a00-b97b-11eb-8094-ece09c018c8a.png)